src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1795,9 +1804,6 @@ private void HandleHostError(Exception exception)
                 // Also notify the invoker so the error can also be written to the function
                 // log file
                 NotifyInvoker(functionException.MethodName, functionException);
-
-                // Mark the error as handled so execution will continue with this function disabled
-                functionException.Handled = true;
```

Comment:
No longer need to set this - the host level knob subsumes this.

Reply:
I'm assuming we already had some tests validating this, and they just continue passing?

Reply:
Yes. However I just added a bit to that test to verify the background retries.

This test was a ServiceBus listener test, and it turned out that the retries exposed a bug in ServiceBusListener state management. See supporting PR https://github.com/Azure/azure-webjobs-sdk/pull/1650/files